tools/packaging/packaging.psm1

```diff
@@ -3109,9 +3110,10 @@ function New-DotnetSdkContainerFxdPackage {
     Write-Log "fxdPackagePath: $FxdPackagePath"
 
     $packageName = "powershell-$Version-$packageNamePlatform-fxd-dotnetsdk$packageNameExtension"
+    $destinationPackageFullName = Join-Path $DestinationPath $packageName
 
     ## Get fxdependent package path
-    $fxdPackage = (Get-ChildItem $FxdPackagePath -Recurse -Filter $basePackagePattern).FullName
+    $fxdPackage = (Get-ChildItem $FxdPackagePath -Recurse -Filter $basePackagePattern).FullName | Select-Object -First 1
```

Comment:
NIT:  I always find this odd.  Seems more logical to use `-ExpandProperty` to me

Reply:
Fixed.